src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/StorePal.Windows.Export.cs

```diff
@@ -75,9 +75,7 @@ public void MoveTo(X509Certificate2Collection collection)
                     }
 
                 case X509ContentType.Pkcs12:
-                    byte[] pkcs12 = ExportPkcs12(false, password, out bool aes256Sha256);
-                    Debug.Assert(!aes256Sha256);
-                    return pkcs12;
+                    return ExportPkcs12Core(null, password);
```

Comment:
`null` means "Don't pass in `PKCS12_EXPORT_PBES2_PARAMS`, and don't re-encode it as anything. Return it as-is", which is the behavior we want for the "old" `Export` API.